### PR TITLE
chore: Add Optimize E2E tests

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #268.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by tofumatt** [comment]
@googlebot I consent

**Comment by tofumatt** [comment]
@googlebot I consent.

**Comment by aaemnnosttv** [comment]
@googlebot I consent.

File: tests/e2e/specs/modules/optimize/activation.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	} );
+
+	it( 'prompts to insert your Optimize ID when Analytics snippet is enabled', async () => {
+		await setupAnalytics( { useSnippet: true } );
+		await proceedToOptimizeSetup();
+
+		const setupHandle = await page.$( '.googlesitekit-setup-module--optimize' );
+		await expect( setupHandle ).toMatchElement( '.googlesitekit-setup-module__title', { text: /Optimize/i } );
+		await expect( setupHandle ).toMatchElement( 'p', { text: /Please copy and paste your Optimize ID to complete your setup/i } );
+		// Not able to use negation here for some reason.
```
Comment (swissspidy) [inline tests/e2e/specs/modules/optimize/activation.test.js]: Really odd. `analyticsUseSnippet` is `true` in the component, so these strings are not rendered. Still, these matchers indeed are not passing.

Would be good to investigate this in a separate ticket in order to not waste time doing that now.

**Pull request merged**